frog/imports/ui/LandingPage/index.js

```diff
@@ -137,39 +148,52 @@ const LandingPage = ({ history }) => {
           complete control over the progress of the class and all it takes is
           three steps.
         </Typography>
-        <Button
-          disableFocusRipple
-          disableRipple
-          className={classes.lineButton}
-          onClick={() => history.push('/wizard')}
-        >
-          Try it out now
-        </Button>
+        <div className={classes.alignCenterDiv}>
+          <Button
+            disableFocusRipple
+            disableRipple
+            className={classes.lineButton}
+            onClick={() => history.push('/wizard')}
```

Comment:
Consider moving `wizard` to a constant as it is used multiple time in this file

Reply:
The other links have sub routes after '/wizard'. Would it be a good idea to replace it? As any change would probably affect the sub routes as well unless we're changing /wizard only to say /activity and the url after that remains unchanged. Let me know what you think.